config/routes.rb

```diff
@@ -1,7 +1,7 @@
 Hubstats::Engine.routes.draw do
   root to: "pull_requests#index" #sets default root to be the pulls page
   post "/handler" => "events#handler", :as => :handler
-  resources :deploys, :only => [:index, :create] #routes to both index and to create method
+  resources :deploys, :only => [:create, :index] #routes to both index and to create method
```

Comment:
I don't believe so in a negative way. I did see a change with one test that was failing, but it wasn't a negative change.